shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1040,6 +1064,23 @@ void WebContents::WebContentsCreatedWithFullParams(
   new WebContentsPreferences(new_contents, dict);
 }
 
+// needed?
+void WebContents::OnPrepareWebContentsCreation(
+    content::WebContents::CreateParams& contentsCreateParams,
+    const content::mojom::CreateNewWindowParams& windowCreateParams) {
+  // HACK: Until electron PR lands to properly pass the right values
```

Comment:
It looks like the PR referenced here was merged. Is this still needed?

Reply:
@alexciarlillo it may not be, but I need to do a bunch of testing here before removing. This is PR represent the initial non-conflicted merge. I'm imagining I'll tweak a couple of thinks before merging

Reply:
Okay. One more question - why target 15.3.2 here instead of 16.0.0? Just hoping for more stability on a longer lived major release?

Reply:
@alexciarlillo yeah that's part of it, I didn't wanna jump to latest in case there are un-forseen/lingering bugs, which would put us in a holding pattern. 15-x-y felt like a good step, given how many version we're jumping.